### PR TITLE
Fix dependency conflict and wrong badge link.

### PR DESCRIPTION
**Pull request by Guillemdb**
- Changing the order of requests and dulwich avoids installing an uncompatible version of urllib3. This happens because requets has a closed interval of valid versions of urllib3 ('urllib3>=1.21.1,<1.25') and dulwich only needs `urllib3[secure]>=1.23`. If dulwich is installed before requests. the installed version of urllb3 will be 1.25, and will make requests fail.

- The project was moved to travis.com from travis.org, so I updated the links of the badge to point to the new url.

### CODE REVIEW

**Comment by Guillemdb** [comment]
@vmarkovtsev Is this coverage fail my fault?

File: requirements.txt [hunk 1]
```diff
@@ -4,11 +4,12 @@ numpy==1.14.0
 scipy==1.0.0
 clint==0.5.1
 google-cloud-storage==1.2.0
+# Requests should be first to avoid dependency problems on urllib3
```
Comment (vmarkovtsev) [inline requirements.txt]:
```suggestion
# requests should be before dulwich to avoid urllib3 version conflict
```

File: setup.py [hunk 2]
```diff
@@ -31,11 +31,11 @@
                       "scipy>=1.0,<2.0",
                       "clint>=0.5.0,<0.6",
                       "google-cloud-storage>=1.2,<=1.2.0",
+                      "requests >=2.0,<3.0",
```
Comment (vmarkovtsev) [inline setup.py]: Commend here is needed too

**Comment by Guillemdb** [comment]
Done, now the message is displayed both in setup.py and requirements.txt

**Pull request merged**